NOTICES.md

```diff
@@ -2,17 +2,17 @@ List of third-party dependencies grouped by their license type
 
 ### [Apache Software License 2.0](./licenses/apache_software_license_2.0.txt):
 * Java Faker ([com.github.javafaker:javafaker:1.0.1](http://github.com/DiUS/java-faker))
-* eureka-client ([com.netflix.eureka:eureka-client:1.9.13](https://github.com/Netflix/eureka))
 * Spring Boot Starter ([org.springframework.boot:spring-boot-starter:2.2.0.RELEASE](https://projects.spring.io/spring-boot/#/spring-boot-parent/spring-boot-starters/spring-boot-starter))
 * Spring Boot Web Starter ([org.springframework.boot:spring-boot-starter-web:2.2.0.RELEASE](https://projects.spring.io/spring-boot/#/spring-boot-parent/spring-boot-starters/spring-boot-starter-web))
 * Spring Cloud Config Client ([org.springframework.cloud:spring-cloud-config-client:2.2.0.RELEASE](https://spring.io))
 * Spring Cloud Netflix Eureka Client ([org.springframework.cloud:spring-cloud-netflix-eureka-client:2.2.0.RELEASE](https://spring.io/spring-cloud/spring-cloud-netflix/spring-cloud-netflix-eureka-client))
+* Spring Cloud Starter Netflix Eureka Client ([org.springframework.cloud:spring-cloud-starter-netflix-eureka-client:2.2.0.RELEASE](https://projects.spring.io/spring-cloud))
 * Spring Cloud Starter OpenFeign ([org.springframework.cloud:spring-cloud-starter-openfeign:2.2.0.RELEASE](https://projects.spring.io/spring-cloud))
-* clients-simple ([uk.gov.gchq.palisade:clients-simple:0.4.0-SNAPSHOT](https://github.com/gchq/Palisade-clients/tree/develop/clients-simple))
-* common ([uk.gov.gchq.palisade:common:0.4.0-SNAPSHOT](https://github.com/gchq/Palisade-common))
-* example-library ([uk.gov.gchq.palisade:example-library:0.4.0-SNAPSHOT](https://github.com/gchq/Palisade-examples/tree/develop/example-library))
-* hr-data-generator ([uk.gov.gchq.palisade:hr-data-generator:0.4.0-SNAPSHOT](https://github.com/gchq/Palisade-examples/tree/develop/hr-data-generator))
-* readers-common ([uk.gov.gchq.palisade:readers-common:0.4.0-SNAPSHOT](https://github.com/gchq/Palisade-readers/tree/develop/readers-common))
+* clients-simple ([uk.gov.gchq.palisade:clients-simple:0.4.0-20200610.163556-44](https://projects.spring.io/spring-boot/#/spring-boot-starter-parent/clients/clients-simple))
+* common ([uk.gov.gchq.palisade:common:0.4.0-SNAPSHOT](no url defined))
+* example-library ([uk.gov.gchq.palisade:example-library:0.4.0-20200615.163811-25](https://github.com/gchq/Palisade-examples/tree/develop/example-library))
+* hr-data-generator ([uk.gov.gchq.palisade:hr-data-generator:0.4.0-20200615.163809-25](https://github.com/gchq/Palisade-examples/tree/develop/hr-data-generator))
```

Comment:
Can you change these to use the SNAPSHOT version. I have raised a bug to fix how the Notices are generated